libs/common/src/types/types-helper.ts

```diff
@@ -0,0 +1,7 @@
+export type ValuesOf<T> = T[keyof T];
+
+// https://www.totaltypescript.com/concepts/the-prettify-helper
+export type Prettify<T> = {
+  [K in keyof T]: T[K];
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Can we remove inline script please?

Reply:
I think this would be a good case for an inline rule because it's an exception rather than a rule we'd want to follow elsewhere. We use `{}` to follow the type mentioned in the in the referenced link, but would want this error to still come up if we tried to use it in our own code elsewhere.

Reply:
As discussed, will remove from here for now and review use on main.